packages/devtools_app/lib/src/charts/flutter/flame_chart.dart

```diff
@@ -175,44 +182,67 @@ class _ScrollingFlameChartRowState extends State<ScrollingFlameChartRow>
         width: widget.width,
       );
     }
-    // Having each row handle gestures instead of each node handling its own
-    // gestures improves performance.
-    return GestureDetector(
-      behavior: HitTestBehavior.opaque,
-      onTapUp: (details) => _handleTapUp(details, context),
-      child: SizedBox(
-        height: rowHeightWithPadding,
-        width: widget.width,
-        child: ListView.builder(
-          addAutomaticKeepAlives: false,
-          // The flame chart nodes are inexpensive to paint, so removing the
-          // repaint boundary improves efficiency.
-          addRepaintBoundaries: false,
-          controller: scrollController,
-          scrollDirection: Axis.horizontal,
-          itemCount: nodes.length,
-          itemBuilder: (context, index) {
-            final node = nodes[index];
-            final nextNode =
-                index == nodes.length - 1 ? null : nodes[index + 1];
-            final paddingLeft = index == 0 ? node.rect.left : 0.0;
-            final paddingRight = nextNode == null
-                ? widget.width - node.rect.right
-                : nextNode.rect.left - node.rect.right;
-            return Padding(
-              padding: EdgeInsets.only(
-                left: paddingLeft,
-                right: paddingRight,
-                bottom: rowPadding,
-              ),
-              child: node.buildWidget(node.data == widget.selected),
-            );
-          },
+    // Having each row handle gestures and mouse events instead of each node
+    // handling its own improves performance.
+    return MouseRegion(
+      onHover: _handleMouseHover,
+      child: GestureDetector(
+        behavior: HitTestBehavior.opaque,
+        onTapUp: (details) => _handleTapUp(details, context),
```

Comment:
Note: _handleTapUp can be a tearoff.  Context is an instance variable on state.

Reply:
done